agency_api/urls.py

```diff
@@ -26,6 +27,7 @@
 router.register('api/caretaker_requests', CareTakerRequestViewSet, 'care-taker-requests')
 router.register('api/create_service_requests', CreateServiceRequestViewSet, 'create-service-requests')
 router.register('api/retrieve_service_requests', RetrieveServiceRequestViewSet, 'retrieve-service-requests')
+router.register('api/job_advertisement_request',HPJobApplicationViewSet, 'hp-job-application')
```

Comment:
Can we change this URL to something that is more in line with the functionality of the end points? The HPJobApplicationViewSet isn't really doing anything with the job advertisements, it's more about the job applications. Could change it to something like "api/job_applications"

Reply:
Also if we update the URL here, we just need to make sure to also update the front end code that makes this API request